exchanges/btcmarkets/btcmarkets_test.go

```diff
@@ -154,9 +154,9 @@ func TestGetFundingHistory(t *testing.T) {
 }
 
 func TestModifyOrder(t *testing.T) {
-	_, err := b.ModifyOrder(1337, exchange.ModifyOrder{})
-	if err == nil {
-		t.Error("Test failed - ModifyOrder() error", err)
+	resp := b.ModifyOrder(exchange.ModifyOrder{})
```

Comment:
This seems to be the only test for modify order where the endpoint is unsupported/Not implemented. I'd say to either add the same tests to all exchanges where its unsupported, or remove this test.

Reply:
Will do